lib/crypto.js

```diff
@@ -0,0 +1,9 @@
+import { createHash } from 'node:crypto'
+
+export function hashEmail ({
+  email,
+  salt = process.env.EMAIL_SALT
```

Comment:
Default to the salt from env, but in theory we can accept others, if that ever becomes useful